brain/neuron/section.h

```diff
@@ -122,7 +122,9 @@ class Section
     BRAIN_API bool hasParent() const;
 
     /**
-     * Return the parent section of this section.
+     * Return the parent section of this section \if pybind or None if doesn't
+     * have any\endif.
+     *
      * @throw runtime_error is the section doesn't have a parent.
```

Comment:
The throw is parsed, but I've just realized that this function doesn't throw in Python.